src/apps/concepts/pages/CreateOrEditConceptPage.tsx

```diff
@@ -120,6 +123,10 @@ const CreateOrEditConceptPage: React.FC<Props> = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  useEffect(() => {
+    return () => { resetConceptForm(); }
+}, []);
```

Comment:
```suggestion
  useEffect(() => resetConceptForm(),
   // eslint-disable-next-line react-hooks/exhaustive-deps
   []);
```

This is a slightly simpler definition, and we should disable the linter for this line as we are intentionally not putting anything in the dependency array.